.github/workflows/code-formatting.yml

```diff
@@ -15,68 +16,69 @@ jobs:
         # diverged. We're fetching everything here, as we don't know how many
         # commits back that point is.
         fetch-depth: 0
+
+    - name: Install prerequisites
+      env:
+        DEBIAN_FRONTEND: noninteractive
+      run: |
+        sudo apt update -y
+        sudo apt install -y clang-format-11
+
     - name: Run clang format
       id: clang_format
       env:
         ALIBUILD_GITHUB_TOKEN: ${{secrets.ALIBUILD_GITHUB_TOKEN}}
       run: |
         set -x
         # We need to fetch the other commit.
-        git fetch origin ${{ github.event.pull_request.base.ref }}
-        git fetch origin pull/${{ github.event.pull_request.number }}/head:${{ github.event.pull_request.head.ref }}
+        git fetch origin ${{ github.event.pull_request.base.ref }} \
+                         pull/${{ github.event.pull_request.number }}/head:${{ github.event.pull_request.head.ref }}
 
         # We create a new branch which we will use for the eventual PR.
         git config --global user.email "alibuild@cern.ch"
         git config --global user.name "ALICE Action Bot"
         git checkout -b alibot-cleanup-${{ github.event.pull_request.number }} ${{ github.event.pull_request.head.sha }}
 
+        echo "Running clang-format-11 against branch ${{ github.event.pull_request.base.ref }}, with hash ${{ github.event.pull_request.base.sha }}"
         # github.event.pull_request.base.sha is the latest commit on the branch
         # the PR will be merged into, NOT the commit this PR derives from! For
         # that, we need to find the latest common ancestor between the PR and
         # the branch we are merging into.
         BASE_COMMIT=$(git merge-base HEAD ${{ github.event.pull_request.base.sha }})
-        echo "Running clang-format-8 against branch ${{ github.event.pull_request.base.ref }} , with hash ${{ github.event.pull_request.base.sha }}"
         COMMIT_FILES=$(git diff --name-only $BASE_COMMIT | grep -ivE 'LinkDef|Utilities/PCG/')
-        [ "X$COMMIT_FILES" = X ] && { echo "No files to check" ; exit 0; }
-        RESULT_OUTPUT="$(git-clang-format-8 --commit $BASE_COMMIT --diff --binary `which clang-format-8` $COMMIT_FILES)"
+        if [ -n "$COMMIT_FILES" ]; then
```

Comment:
Do you really need to change the rest of the logic to bump clang-format 11?

Reply:
OK, I've undone most of the formatting changes. Is this better?

Reply:
yes. Could you move the `11` in a variable?

Reply:
Alternatively, can't we simply drop the `-11`?

Reply:
I've put the -11 in a variable. If we drop the -11, it'll use clang-format-10 in the Ubuntu containers provided by GitHub.

Reply:
Sorry for being pedantic... Actually I think:

```
update-alternatives --install /usr/bin/clang-format clang-format /usr/bin/clang-format-11 100
```

is better. This way we do not need a custom environment variable and we can simply have `clang-format`. Am I missing something?

Reply:
OK, sure! I've also redirected git-clang-format and removed the `--binary ...` options as they should be superfluous now.